frontend/src/app/admin-details-page/admin-details-page.component.ts

```diff
@@ -22,7 +23,8 @@ export class AdminDetailsPageComponent implements OnInit {
 
   constructor(private adminService: AdminService,
               private route: ActivatedRoute,
-              private notificationsService: NotificationsService) {
+              private notificationsService: NotificationsService,
+              private translateService: TranslateService) {
```

Comment:
Wird gar nicht verwendet? Und weiter unten werden doch die Strings in den notificationsService gebuttert?